app/controllers/account/dashboard_controller.rb

```diff
@@ -5,15 +5,28 @@ def index
   end
 
   def top_workspaces_card
-
   end
-  
-  def user_info_card
 
-  end
+  def tasks_info
+    @task_info = []
```

Comment:
move logic from controller to new class

app/lib/TaskInfoReport and pass current_user and return hash

and in controller

render json: { info: TaskInfo.new(current_user).report, workspaces: WorkspacesReport.new(current_user).report }